project/SparkBuild.scala

```diff
@@ -1075,7 +1075,6 @@ object ExcludedDependencies {
     // purpose only. Here we exclude them from the whole project scope and add them w/ yarn only.
     excludeDependencies ++= Seq(
       ExclusionRule(organization = "com.sun.jersey"),
-      ExclusionRule("javax.servlet", "javax.servlet-api"),
```

Comment:
this is intended, to avoid conflict with `jakarta.servlet-api:4.x`, the tricky thing is that `jakarta.servlet-api:4.x` actually contains `javax.servlet` classes

Reply:
Ya, this is a spin-off from the original PR. We know that this is a transient status, @pan3793 . Do you have any other concern?

Reply:
I get the point. The change makes sense to me.